canvas_api_client/tests/test_v1_client.py

```diff
@@ -61,7 +56,7 @@ def test_send_request(self):
                 'x': 'y'
             })
 
-        returned_response.raise_for_status.assert_called_once()
```

Comment:
so strange that this wasn't working on travis. glad it's working with the new way though :)

Reply:
It's missing the `_with` at the end!

https://engineeringblog.yelp.com/2015/02/assert_called_once-threat-or-menace.html

Reply:
Subtle, insidious bug.